t/unit-tests/t-reftable-pq.c

```diff
@@ -18,6 +18,11 @@ static void merged_iter_pqueue_check(struct merged_iter_pqueue *pq)
 	}
 }
 
+static int pq_equal(struct pq_entry *a, struct pq_entry *b)
```

Comment:
Let's rename this to `pq_entry_equal` as you are not checking whether two priority queues are equal.